build/package/Dockerfile

```diff
@@ -42,9 +42,9 @@ LABEL authors="Julien Neuhart <j.neuhart@thecodingmachine.com>"
 
 ARG TINI_VERSION
 
-ADD https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
-ENTRYPOINT [ "/tini", "--" ]
+ADD https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini-static /usr/local/bin/tini
```

Comment:
What is the difference between `tini` and `tini-static`?

Reply:
It was a try to solve Cloud Run issue, related to this: https://github.com/krallin/tini#statically-linked-version

But it didn't work. Although it is safer to use statically linked binaries, is you want I can revert this change!

Reply:
Alright, let's keep it this way then 👍